nixos/tests/kanidm.nix

```diff
@@ -71,5 +77,7 @@ import ./make-test-python.nix ({ pkgs, ... }:
         client.wait_until_succeeds("kanidm login -D anonymous && kanidm self whoami | grep anonymous@${serverDomain}")
```

Comment:
What is the reason this is a `wait_until_succeeds` instead of a `succeed`? In previous commands it is already established the service is up and responds right?

Reply:
Good question, that is probably my code. I'm don't remember any reasons for it. Did you try running it with `succeed`?

Reply:
Yeah, no issues. Want me to change L76&77 to `succeed` in another commit?

Reply:
Sure!

Reply:
I mean there is no necessity for a second commit, just do as you prefer.